README.md

```diff
@@ -1,159 +1,45 @@
-<img src=".erb/img/erb-banner.svg" width="100%" />
+# Notes App
 
-<br>
+This project is a notes app built using React and Electron. It allows users to create, organize, and search through their notes efficiently. The app features a simple, user-friendly interface designed for productivity and note management.
```

Comment:
> It allows users to create, organize, and search through their notes efficiently.

Remove this sentence as we are not promising anything as of now.

Reply:
I have provided a brief description of the app in the recent commit